activesupport/lib/active_support/testing/method_call_assertions.rb

```diff
@@ -30,22 +30,8 @@ def assert_not_called(object, method_name, message = nil, &block)
           assert_called(object, method_name, message, times: 0, &block)
         end
 
-        #--
-        # This method is a temporary wrapper for mock.expect as part of
-        # the Minitest 5.16 / Ruby 3.0 kwargs transition. It can go away
-        # when we drop support for Ruby 2.7.
-        if Minitest::Mock.instance_method(:expect).parameters.map(&:first).include?(:keyrest)
-          def expect_called_with(mock, args, returns: false, **kwargs)
-            mock.expect(:call, returns, args, **kwargs)
-          end
-        else
-          def expect_called_with(mock, args, returns: false, **kwargs)
-            if !kwargs.empty?
-              mock.expect(:call, returns, [*args, kwargs])
-            else
-              mock.expect(:call, returns, args)
-            end
-          end
```

Comment:
@eileencodes as the author of the [original commit](https://github.com/rails/rails/blame/a73cbbea6a082cdd320ecc92e3c6f1d46fe6af74/activesupport/lib/active_support/testing/method_call_assertions.rb#L33-L50), I'm curious: have I picked the correct fork of the conditional to remove?